equinix/resource_metal_virtual_circuit_acc_test.go

```diff
@@ -129,8 +129,9 @@ func TestAccMetalVirtualCircuit_dedicated(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{ // Error: Error waiting for virtual circuit 863d4df5-b3ea-46ee-8497-858cb0cbfcb9 to be created: GET https://api.equinix.com/metal/v1/virtual-circuits/863d4df5-b3ea-46ee-8497-858cb0cbfcb9?include=project%2Cport%2Cvirtual_network%2Cvrf: 500 Oh snap, something went wrong! We've logged the error and will take a look - please reach out to us if you continue having trouble.
 		PreCheck:          func() { testAccPreCheck(t) },
 		ExternalProviders: testExternalProviders,
-		ProviderFactories: testAccProviderFactories,
-		CheckDestroy:      testAccMetalVirtualCircuitCheckDestroyed,
+		//ProviderFactories:        testAccProviderFactories,
+		ProtoV5ProviderFactories: testAccProtoV5ProviderFactories,
```

Comment:
This fix enabled the test to use the metal_vlan resource, but it now fails with a different error:
```
    resource_metal_virtual_circuit_acc_test.go:129: Step 1/3 error: Error running apply: exit status 1
        
        Error: Value Conversion Error
        
          with equinix_metal_vlan.test,
          on terraform_plugin_test.tf line 28, in resource "equinix_metal_vlan" "test":
          28:         resource "equinix_metal_vlan" "test" {
        
        An unexpected error was encountered trying to convert tftypes.Value into
        vlans.DataSourceModel. This is always an error in the provider. Please report
        the following to the provider developer:
        
        mismatch between struct and object: Struct defines fields not found in
        object: vlan_id.
```